### PR TITLE
Remove code tags from challenge seed

### LINKED ISSUE #13181: remove <code> tags
Challenge [enhanced-object-literals--functions](http://beta.freecodecamp.com/en/challenges/es6/enhanced-object-literals--functions) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/56.0.2924.87 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

There are some \<code\> tags in the tests at the bottom. Prevents one to pass the test. Works fine if removed. 

```js
/* Alter code above this line */
<code>const zod = Person("Zodiac Hasbro", 56, 'male');</code>
<code>const yan = Person("Yanoshi Mimoto", 55, 'male');</code>
<code>zod.sendFriendRequest(yan);</code>

```

### PR DESCRIPTION
**Pull request by Manish-Giri**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #13181 

#### Description

Removed code tags from the challenge seed, which were causing some tests to fail. Also added code tags around the ES6 function syntax in `description`, to be consistent with the example above it.

--- 
UPDATE - 

Improved the code formatting in `description`, as per Eric's suggestions. Here's the latest changes - 

screenshot 2017-02-07 00 25 37

Closes #13181

### CODE REVIEW

File: seed/challenges/02-javascript-algorithms-and-data-structures/es6.json [hunk 1]
```diff
@@ -660,13 +660,13 @@
...
         "<code>}</code>",
         "We can remove the function keyword and colon (:) altogether - and get this:",
-        " const Container extends Component {",
-        "     render() {",
-        "       return {",
-        "         `<div>Container</div>`",
-        "       }",
-        "     }",
-        " }",
+        "<code> const Container extends Component { </code>",
```
Comment (erictleung) [inline seed/challenges/02-javascript-algorithms-and-data-structures/es6.json]: This multi-line example in the description of the challenge should be in `<blockquote>` tags. [Here's](https://github.com/freeCodeCamp/freeCodeCamp/blob/15474e8065d633657842a3208b1e6ca34814abf6/seed/challenges/02-javascript-algorithms-and-data-structures/object-oriented-programming.json#L45) an example of it being used. You can use `<br>` for line breaks. Make sure those spaces are there to retain the formatting.

**Review by erictleung — CHANGES_REQUESTED** [review]

File: seed/challenges/02-javascript-algorithms-and-data-structures/es6.json (hunk 1 above)
Comment (Manish-Giri) [inline seed/challenges/02-javascript-algorithms-and-data-structures/es6.json]: @erictleung So, I tried out the `<blockquote>` on the first code example. Please review this, I can move on to the next example if this one looks all right - 

screenshot 2017-02-06 18 58 43

Comment (erictleung) [inline seed/challenges/02-javascript-algorithms-and-data-structures/es6.json]: @Manish-Giri yep, the top part is what it should look like!

**Comment by raisedadead** [comment]
@erictleung this LGTM, unless you want to add anything more to the change.

**Comment by erictleung** [comment]
@raisedadead if you're able to locally test this, go ahead and merge. I haven't made the time yet to get to this and pull it down locally.

**Review by erictleung — APPROVED** [review]

**Pull request merged**